ui/js/compareperf.js

```diff
@@ -22,18 +22,20 @@ perf.controller('CompareChooserCtrl', [
         });
         
         ThResultSetModel.getResultSetsFromRevision($scope.newProject.name, $scope.newRevision).then(
-              function(resultSets) {
-               $scope.newRevisionError = undefined;
-               if($scope.originalRevisionError == undefined && $scope.newRevisionError == undefined) {
-                $state.go('compare', {
-                originalProject: $scope.originalProject.name,
-                originalRevision: $scope.originalRevision,
-                newProject: $scope.newProject.name,
-                newRevision: $scope.newRevision });        
+            function (resultSets) {
+              $scope.newRevisionError = undefined;
```

Comment:
I believe this should be also indented 4 spaces, not 2. So it is aligned with the if statement below it.